General.py

```diff
@@ -0,0 +1,41 @@
+def sumEvenNumbers(arr):
+    '''accepts an array of integers and returns the sum of all the even numbers in the array
+        ARGS - arr: array of integers
+        RETURNS- sum of all even numbers: int'''
+    return sum(filter(lambda x: x%2 == 0,arr))
+
+def commonElements(arr1,arr2):
+    '''function that accepts two arrays and returns a new array that contains only the elements that are common between the two arrays'''
+    return list(set(arr1).intersection(set(arr2)))
+
+def secondSmallestInArray(arr):

```

Comment:
Perfect implementation

Reply:
You would actually need to make it a set(arr) first before you sort as the array [1,1,2,3] we would want to return 2 not 1